server/game/cards/09.3-TSs/GrimoiresAndMore.js

```diff
@@ -0,0 +1,50 @@
+const DeedCard = require('../../deedcard.js');
+const GameActions = require('../../GameActions/index.js');
+const StandardActions = require('../../PlayActions/StandardActions.js');
+
+class GrimoiresAndMore extends DeedCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'React: Grimoires & More',
+            when: {
+                onCardEntersPlay: event => event.card.controller === this.controller && event.card.getType() === 'dude' && event.card.hasKeyword('huckster')
+            },
+            cost: ability.costs.bootSelf(),
+            handler: context => {
+                const dude = context.event.card;
+                context.player.pull((pulledCard, pulledValue, pulledSuit) => {
+                    if('clubs' === pulledSuit.toLowerCase()) {
+                        this.game.resolveGameAction(GameActions.sendHome({ card: dude, needToBoot: true}), context);
```

Comment:
Last thing (sorry, did not notice before). There also should be a message here that Grimoires has been used but huckster was sent home due to clubs pull.

Reply:
good catch, i'll add that message over lunch today.

Reply:
:heavy_check_mark:

Reply:
Do not want to be a pain 😄 But that message seems like the deed was used to send the huckster home, and not that it has failed (pulled clubs) and was sent home because of that. I would format is similarly to other experimental cards:
> {0} uses {1} to pull {2} and has to send {3} home because it is a club
or something like that. What do you think?

Reply:
that's much better. will add it later tonight. thanks!

Reply:
:heavy_check_mark:

Reply:
ready for re-review.